src/docker/itk-wasm-base/Dockerfile

```diff
@@ -7,14 +7,22 @@ LABEL org.opencontainers.image.source="https://github.com/InsightSoftwareConsort
 
 WORKDIR /
 
-# 2022-07-02 master branch
-ENV ITK_GIT_TAG 7e35b6443f7bec2aa74d439d066881971f5590db
-RUN git clone https://github.com/thewtex/ITK.git && \
+# 2022-07-02 branch: ITKWASM-2022-07-02-5e7aea9
+ENV ITK_GIT_TAG afd7a44bd42cb56edb8fcd67bd0f5ecaa2e0f3db
+RUN git clone https://github.com/KitwareMedical/ITK.git && \
   cd ITK && \
   git checkout ${ITK_GIT_TAG} && \
   cd ../ && \
   sed -i -e '/^option(OPJ_USE_THREAD/c\option(OPJ_USE_THREAD "use threads" OFF)' \
     /ITK/Modules/ThirdParty/GDCM/src/gdcm/Utilities/gdcmopenjpeg/src/lib/openjp2/CMakeLists.txt
+
+# Modify CMake variable to use patched DCMTK library
+RUN sed -i -e '/^set(DCMTK_GIT_REPOSITORY/c\set(DCMTK_GIT_REPOSITORY "https://github.com/jadh4v/DCMTK.git")' \
```

Comment:
Could you please push this to a branch here:

https://github.com/InsightSoftwareConsortium/DCMTK

Reply:
Done, check if you like the branch name or if it should be different.
Branch name is: 20220802_DCMTK_PATCHES_FOR_ITK-wasm